netweaver/pre_validation.sls

```diff
@@ -0,0 +1,26 @@
+{% from "netweaver/map.jinja" import netweaver with context -%}
+
+{% set host = grains['host'] %}
+
+{# Check for SWPM archive media checkbox #}
+{% if netweaver.use_swpm_sar_file is defined and netweaver.use_swpm_sar_file == false %}
+    {% do netweaver.pop('swpm_sar_file', none) %}
+{% endif %}
+
+{% for node in netweaver.nodes if node.host == host %}
+
+    {# Check for ascs_virtual_host checkbox #}
+    {% if node.ascs_virtual_host_checkbox is defined and node.ascs_virtual_host_checkbox == false %}
```

Comment:
I don't understand this. Why `ascs`?
What happens for the other types (`ers, pas, aas`)?

Reply:
I checked the installation states, the `ascs_virtual_host` entry is needed for `pas` installation. Is this correct?
https://github.com/SUSE/sapnwbootstrap-formula/blob/84ced2387dfdf481ffab0c8eea37767340ecf7e1/netweaver/install_pas.sls#L32
https://github.com/SUSE/sapnwbootstrap-formula/blob/84ced2387dfdf481ffab0c8eea37767340ecf7e1/pillar.example#L138

So i have asked it in the form. It is also added in pillar.example for db isntance.. but not used for installation? Is the pillar example correct or should the  `ascs_virtual_host` be removed from `db` instance entry?

Reply:
Ok, now I understand.